### PR TITLE
Change controller behaviour depending whether rack is added or edited

### PR DESCRIPTION
**Pull request by damian1976**
Change the message from "Rack has been successfully added" in case when the rack is edited...

### CODE REVIEW

File: src/ralph/dc_view/static/js/app/data_center/controllers.js [hunk 1]
```diff
@@ -59,7 +59,12 @@
                         rack.new = false;
                         rack.saved = true;
                         $scope.forms.edit_form.$error.all = null;
-                        $scope.forms.edit_form.$success = ['The rack has been successfully added to data center.'];
+                        if (rack.id !== undefined) {
```
Comment (vi4m) [inline src/ralph/dc_view/static/js/app/data_center/controllers.js]: if typeof(rack.id) !== "undefined"

Comment (damian1976) [inline src/ralph/dc_view/static/js/app/data_center/controllers.js]: OK. If so, the line no. 47 (original) should also be changed in the same way (?)

Comment (vi4m) [inline src/ralph/dc_view/static/js/app/data_center/controllers.js]: Congrats - you've found a potential mistake - you can fix it as well :)

**Comment by ar4s** [comment]
:ok:

**Comment by mkurek** [comment]
:ok:

**Pull request merged**